config/management/operational/Cargo.toml

```diff
@@ -40,6 +40,7 @@ aptos-temppath = { path = "../../../crates/aptos-temppath" }
 fallible = { path = "../../../crates/fallible" }
 netcore = { path = "../../../network/netcore" }
 network = { path = "../../../network" }
+move-core-types = { git = "https://github.com/move-language/move", rev = "f2e7585b1ed5bd2810163d6bdebafe5a388881d3", features=["address32"] }
```

Comment:
We now use a single move crate: https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-deps